src/scripts/components/PillsInput.jsx

```diff
@@ -50,6 +52,7 @@ class PillsInput extends React.Component {
         {renderPills()}
         <div className="pill-input__input-container">
           <Input
+            format={inputProps.format}
```

Comment:
With this change it doesn't seem the Input will be able to take in any new props, only format, also we still have old props like `naked`, `name` etc. 

The Input component should basically take `...inputProps` as its only argument and all of these are encompassed in that object.